ChemoTracker/components/Calendar/Calendar.js

```diff
@@ -10,16 +10,14 @@ import CalendarEvent from './event.js';
 import _ from 'lodash';
 import moment from 'moment';
 import { getSymptoms, getSymptomsByMonth, getFilePath } from '../../services/symptomTracking.js';
-import { login } from '../../services/login.js';
-
-let token = '';
 
 class Calendar extends Component {
   constructor(props){
     super(props);
     this.state = {
       listOfItems: {},
-      calendarItems: {}
+      calendarItems: {},
+      token: props.screenProps.token
```

Comment:
If I add this to profile, will it still work?

Reply:
yes it will, i mean it should, i just havent added it to profile